generate-code.sh

```diff
@@ -11,7 +11,7 @@
 set -e
 
 echo "~ START"
-PROTOBUF_VERSION=3.19.0
+PROTOBUF_VERSION=21.6
```

Comment:
Should this be
```suggestion
PROTOBUF_VERSION=3.21.6
```

Reply:
Nope. See the commit comment - protobuf has changed its versioning. (This is protobuf project versioning, not NuGet package versioning which we use elsewhere.)